### DISCUSSION TITLE
RFC:Multi hop query results flat sampling

### DISCUSSION THREAD

**Discussion by zhaojunnana** [vesoft-inc/nebula#4789] [General]
In multi hop queries, multi ID sampling is often used
![在这里插入图片描述](https://img-blog.csdnimg.cn/e7ca254af5db4eaf8f1f88413463eefb.png)

It is recommended to add FLATSAMPLE <sample_list> syntax in { SAMPLE <sample_list> | <limit_by_list_clause> }

pr:https://github.com/vesoft-inc/nebula/pull/4797

**Comment by wey-gu** [comment]
ref: https://github.com/zhaojunnana/nebula/blob/master/docs/ngql_push_filter_down.md#ngql%E8%AF%AD%E6%B3%95-sample-sample_list--limit_by_list_clause-%E4%B8%AD%E6%B7%BB%E5%8A%A0flatsample-sample_list%E8%AF%AD%E6%B3%95

**Comment by wey-gu** [comment]
cc @MuYiYong
